app/views/dishes/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1><%= @dish.name %></h1>
+<p><%= @dish.description %></p>
+
+<section class="ingredients">
+  <h3>Ingredients</h3>
+  <% @dish.ingredients.each do | ingredient | %>
+    <section id="ingredient-<%= ingredient.id %>", style="display:inline-block;">
+<!--      <div style="display: inline-block">-->
+        <p><%= ingredient.name %></p>
+        <%= button_to "Remove Ingredient",
+                      dish_ingredient_path(@dish),
+                      method: :delete,
+                      data: {
```

Comment:
You shouldn't need to include this Turbo parameter because Turbo doesn't affect buttons, just FYI!

Reply:
when using the nested resources syntax in routes, I would expect the `dish_ingredient_path` to also include an ingredient id ("/dishes/:dish_id/ingredients/:id") so you should be able to pass the `@dish` and `ingredient` as parameters. Then you don't need the params object either